portal/src/graphql/adminapi/PrimaryIdentitiesSelectionForm.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useCallback, useContext, useMemo } from "react";
+import { Dropdown, PrimaryButton } from "@fluentui/react";
+import { Context, FormattedMessage } from "@oursky/react-messageformat";
+import { useDropdown } from "../../hook/useInput";
+
+import { IdentityLists } from "./UserDetailsConnectedIdentities";
+
+import styles from "./PrimaryIdentitiesSelectionForm.module.scss";
+import deepEqual from "deep-equal";
+import NavigationBlockerDialog from "../../NavigationBlockerDialog";
+
+interface PrimaryIdentitiesSelectionFormProps {
+  className?: string;
+  identityLists: IdentityLists;
+}
+
+const PrimaryIdentitiesSelectionForm: React.FC<PrimaryIdentitiesSelectionFormProps> = function PrimaryIdentitiesSelectionForm(
```

Comment:
FYI we may need to revamp this section after the custom attribute spec is confirmed, as we may have a separate tab to manage it.

cc @louischan-oursky